js/helpers/syllableCountIterator.js

```diff
@@ -19,11 +19,11 @@ var SyllableCountIterator = function( config ) {
 /**
  * Creates a language syllable regex for each exclusion.
  *
- * @param {object} syllableRegexes The object containing all exclusion syllables including the multipliers.
+ * @param {object} syllableCountSteps The object containing all exclusion syllables including the multipliers.
```

Comment:
The name of this variable seems to mean something else than the description. Seeing as the function implies you are creating a SyllableCountSteps object, passing one into it makes little sense.